src/ui/annotations.ts

```diff
@@ -1073,43 +1079,348 @@ export class AnnotationSchemaView extends Tab {
         this.updateProperty(oldProperty, newProperty);
       },
     });
-    this.element.appendChild(updateButton);
+    this.schemaActionButtons.appendChild(updateButton);
 
     const downloadButton = makeIcon({
-      text: "Download schema",
       title: "Download schema",
+      svg: svg_share,
       onClick: () => this.downloadSchema(),
     });
-    this.element.appendChild(downloadButton);
+    this.schemaActionButtons.appendChild(downloadButton);
 
     const copyButton = makeCopyButton({
       title: "Copy schema to clipboard",
       onClick: () => this.copySchemaToClipboard(),
     });
-    this.element.appendChild(copyButton);
+    this.schemaActionButtons.appendChild(copyButton);
 
     const pasteButton = makeIcon({
-      text: "Paste schema from clipboard",
       title: "Paste schema from clipboard",
+      svg: svg_clipboard,
       onClick: () => this.pasteSchemaFromClipboard(),
     });
-    this.element.appendChild(pasteButton);
+    this.schemaActionButtons.appendChild(pasteButton);
+  }
+
+  private createSchemaTable() {
```

Comment:
We don't need to do this right now, but this schema UI logic is getting pretty large, it might be nice if we try to extract as much as we can into `src/ui/annotation_schema.ts`